frontend/src/features/properties/list/buildings/columns.tsx

```diff
@@ -0,0 +1,136 @@
+import { ReactComponent as BuildingSvg } from 'assets/images/icon-business.svg';
+import { ReactComponent as LandSvg } from 'assets/images/icon-lot.svg';
+
+import React from 'react';
+import { CellProps } from 'react-table';
+import { Link } from 'react-router-dom';
+import { formatMoney, formatNumber } from 'utils';
+import { ColumnWithProps } from 'components/Table';
+import { IProperty } from './IProperty';
+
+const MoneyCell = ({ cell: { value } }: CellProps<IProperty, number>) => formatMoney(value);
```

Comment:
I'm going to log a story to centralize some of these column utilities, this code is being duplicated all over the place.